runtime/onert/api/nnfw/src/nnfw_api_internal.cc

```diff
@@ -994,9 +994,9 @@ NNFW_STATUS nnfw_session::set_config(const char *key, const char *value)
 
   const std::string skey = key;
 
-  if (skey == config::TRACE_FILEPATH)
+  if (skey == config::TRACING_MODE)
   {
-    _coptions->trace_filepath = value;
+    _coptions->tracing_mode = value;
```

Comment:
`value` looks like `const char*`, but `_coptions->tracing_mode` looks like boolean. 
Is it okay and intended logic?

Reply:
otherwise looks good to me.

Reply:
You're right. I'll fix this to set boolean value.